src/test/java/com/wikia/webdriver/common/remote/operations/ValidatePost.java

```diff
@@ -0,0 +1,20 @@
+package com.wikia.webdriver.common.remote.operations;
+
+import com.google.common.collect.ImmutableMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.collect.ImmutableMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)